axonserver/src/main/java/io/axoniq/axonserver/logging/AuditLog.java

```diff
@@ -0,0 +1,61 @@
+package io.axoniq.axonserver.logging;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationListener;
+import org.springframework.security.authentication.event.AbstractAuthenticationEvent;
+import org.springframework.security.authentication.event.AbstractAuthenticationFailureEvent;
+import org.springframework.security.authentication.event.AuthenticationSuccessEvent;
+import org.springframework.security.authentication.event.InteractiveAuthenticationSuccessEvent;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Component;
+
+import java.security.Principal;
+
+@Component
+public class AuditLog implements ApplicationListener<AbstractAuthenticationEvent> {
```

Comment:
Don't forget to javadoc this class once it moves from draft to full fledged pull request!